examples/jaeger/src/main/java/io/opentelemetry/example/JaegerExample.java

```diff
@@ -0,0 +1,76 @@
+package io.opentelemetry.example;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+
+public class JaegerExample {
+  // Jaeger Endpoint URL and PORT
+  String ip; // = "jaeger";
+  int port; // = 14250;
+  // OTel API
+  Tracer tracer;
+  // Create a channel towards Jaeger end point
+  ManagedChannel jaegerChannel;
+  // Export traces to Jaeger
+  JaegerGrpcSpanExporter jaegerExporter;
+
+  private void initTracer() {
```

Comment:
Examples will be copied a lot, and because of that I would do the following:
1) Separate "// Give a name to the tracer  this.tracer = tracerFactory.get("io.opentelemetry.example.JaegerExample");" in a different place maybe as a first think in the myWUC
2) Rename initTracer -> setupJaeger (or more suggestive name) and call this function in main();

This way we separate setting the exporter from the instrumentation (getting a named tracer is part of instrumentation).

Reply:
Thank you for comment @bogdandrutu. I changed the method name to `setupJaegerExporter` to further stress the setup of an exporter.